metrics/web/lib/dashboard/presentation/models/dashboard_page_parameters_model.dart

```diff
@@ -0,0 +1,57 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:metrics/common/presentation/navigation/models/page_parameters_model.dart';
+import 'package:metrics/dashboard/presentation/pages/dashboard_page.dart';
+
+/// A class, that represents [DashboardPage] parameters parsed from
+/// query parameters.
+class DashboardPageParametersModel extends PageParametersModel {
+  /// A [String], that represents a name of the project.
+  final String projectFilter;
+
+  /// A [String], that represents an identifier of the selected project group.
+  final String projectGroupId;
+
+  /// Creates a new instance of the [DashboardPageParametersModel]
+  /// with the given parameters.
+  DashboardPageParametersModel({
```

Comment:
🎯 
As the `PageParametersModel` is considered **immutable** (inherited from the `Equatable`), would it be better to introduce the `const` constructor to the `PageParametersModel` and descendants according to [this rule](https://dart.dev/guides/language/effective-dart/design#consider-making-your-constructor-const-if-the-class-supports-it) and [this lint](https://dart-lang.github.io/linter/lints/prefer_const_constructors_in_immutables.html)?
```suggestion
  const DashboardPageParametersModel({
```